secboot/secboot.go

```diff
@@ -71,10 +71,10 @@ type SealKeyRequest struct {
 	BootstrappedContainer BootstrappedContainer
 	// The key name; identical keys should have identical names
 	KeyName string
-
+	// The name of the slot where they key will be saved.
 	SlotName string
-	// The path to store the sealed key file. The same Key/KeyName
-	// can be stored under multiple KeyFile names for safety.
+	// The file to store the key data. If empty, the key data will
+	// be saved to the token.
 	KeyFile string
```

Comment:
shouldn't we use also  KeyFileOrSlotName ?

Reply:
This is a different thing than for `KeyFileOrSlotName`. We always need `SlotName`. Because this is where the key will be in the luks2 container. But the key data might not be stored in the token. `KeyFileOrSlotName` will always have both key file and slot name, but here we always need the slot name, but not always the key file.